tests/drivers/spi/spi_loopback/boards/nucleo_f746zg.conf

```diff
@@ -1,2 +1,10 @@
 CONFIG_SPI_LOOPBACK_DRV_NAME="SPI_1"
-CONFIG_SPI_STM32_INTERRUPT=y
```

Comment:
I'm not sure we need to convert all tests to use DMA, as we won't test IRQ mode anymore in the end

Reply:
Ok to  modify only the existing test cases